zerver/openapi/zulip.yaml

```
@@ -13592,12 +13592,23 @@ paths:
 
                           The fake email domain that will be used for new bots created this
                           organization. Useful for UI for creating bots.
-                      realm_uri:
+                      realm_url:
                         type: string
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
                           The URL for the organization.
+                          **Changes**: New in Zulip 8.0 (feature level 192). The name `realm_url`
+                          is preferred use than `realm_uri` as later is deprecated.
+                      realm_uri:
+                        type: string
+                        deprecated: true
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The URL for the organization. Alias of `realm_url`.
+                          **Changes**: Deprecated in Zulip 8.0 (feature level 192). The name
+                          `uri` is deprecated per the HTML standard (https://url.spec.whatwg.org/#goals).
```

```suggestion
                          **Changes**: Deprecated in Zulip 8.0 (feature level 192).
                          The term "URI" is deprecated in web standards (https://url.spec.whatwg.org/#goals).
```

In particular I prefer to reserve code font (Markdown backticks) for fragments of code, like exact identifiers. Fragments of English, including individual words that are part of how we made up a given identifier, appear in normal non-code font.

Here, there's nothing in sight that's called `uri` at a code level; rather there's an identifier `realm_uri`, and its name is made from the words "realm" and "URI". So "URI" gets non-code font. (And it also gets the capitalization that it normally gets in English prose, instead of the one it got as part of the identifier.)